couchdbkit/client.py

```diff
@@ -285,7 +268,6 @@ def __init__(self, uri, create=False, server=None, **params):
         if create:
             self.cloudant_database.create()
 
-        self.res = server.res(self.dbname)
         self._request_session = self.server._request_session
         self.database_url = self.cloudant_database.database_url
```

Comment:
not related to this PR but why are these named `cloudant_X`?

Reply:
I'm not sure why I didn't go with `database` at the beginning, but in order to move away from restkit (very  hard py3 migration to get correct because it's sending raw bytes over sockets) we moved to cloudant's couchdb library (uses requests and isn't reliant on cloudant specific features).

I think that could be changed to something better now though. I don't remember if there was a particular reason i chose this name, but guessing there was a lot of variable naming around database already

Reply:
Thanks, no need to change now. I was just curious.